src/proto/Cosmos.proto

```diff
@@ -135,6 +135,9 @@ message SigningOutput {
     // Signed transaction in protobuf encoded form, Base64-encoded (Stargate case)
     string serialized = 3;
 
+    // Json format directly suitable for broadcast message, containing Stargate serialized field.
+    string json_serialized = 4;
```

Comment:
I think in all our cases we always return data of transaction to be broadcasted to the network, unless there is other reasons to keep both?

Reply:
yeah, just replace the original `serialized` field, should be fine to change this

Reply:
OK, just replacing contents of the serialized field, but keeping its name.